Makefile

```diff
@@ -83,7 +83,7 @@ all: build
 container: all
 	@echo ">> building container"
 	@cp Dockerfile _output/Dockerfile
-	docker build -t $(IMAGE):$(IMAGE_TAG) ${DBUILD_ARGS} -f _output/Dockerfile _output
+	@sudo docker build -t $(IMAGE):$(IMAGE_TAG) ${DBUILD_ARGS} -f _output/Dockerfile _output
```

Comment:
Are these changes really required? sudo?

Reply:
yes, needed for docker.